### PR TITLE
fix(version): use the version from Cargo.toml instead of the hardcode…

### PR DESCRIPTION
**Pull request by chefsale**
…d one:

```
Before:
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/near-indexer-for-explorer (master)> ./target/debug/indexer-explorer --version
indexer-explorer 0.1

After:
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/near-indexer-for-explorer (master)> cargo build -p indexer-explorer
   Compiling indexer-explorer v0.6.1 (/home/sandi/workspace/near/near-indexer-for-explorer)
    Finished dev [unoptimized + debuginfo] target(s) in 30.33s
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/near-indexer-for-explorer (master)> ./target/debug/indexer-explorer --version
indexer-explorer 0.6.1
```

### CODE REVIEW

**Review by frol — APPROVED** [review]

File: src/configs.rs
Comment (frol) [inline src/configs.rs:10]: This is enough:

```suggestion
    author,
```

clap will take the information from Cargo.toml :smile: (too smart, right?)

Comment (chefsale) [reply src/configs.rs:10]: Too smart, fixed 🥇

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
@chefsale I am just wondering, once we merge PRs to indexer-for-explorer, do we trigger any re-deployment?

Feel free to merge this PR once CI pass

**Comment by chefsale** [comment]
Not, we will trigger it on the next release. I don't think there is need to redeploy just because of this 😄

**Comment by chefsale** [comment]
Technically if you want I could setup the CI to do that as well?

**Pull request merged**
